app/assets/javascripts/alchemy/admin.js

```diff
@@ -37,6 +37,7 @@
 //= require alchemy/alchemy.gui
 //= require alchemy/alchemy.hotkeys
 //= require alchemy/alchemy.i18n
+//= require alchemy/alchemy.internal_links_select
```

Comment:
We should require the file in `alchemy.link_dialog` instead, because it's only used there